### PR TITLE
Support for Flask

### PR DESCRIPTION
**Pull request by quamilek**
This PR provides a integration with Flask apps and enpoints in easy way.

### CODE REVIEW

File: docs/flask.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            'retentionTime': 1,
+            'trackingEnabled': False,
+            'contentType': 'JSON',
+            'validationEnabled': False,
+        }
+    }
+}
+```
+
+> Use `<YOUR-APP-URL>/hermes/events/<TOPIC-NAME>` (for example `http://my-django-app.local/hermes/events/pl.allegro.pyhermes.test1`) to subscribe to particular topic in Hermes.
```
Comment (mkurek) [inline docs/flask.md]: my-flask-app

File: docs/flask.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+## Test command
+After instalation you can test your configuration by following command:
+
+```bash
+./manage.py hermes test
```
Comment (mkurek) [inline docs/flask.md]: there is no `manage.py` in flask :)

File: docs/flask.md (hunk 1 above)
Comment (quamilek) [reply docs/flask.md]: done

File: docs/flask.md (hunk 2 above)
Comment (quamilek) [reply docs/flask.md]: done

**Comment by mkurek** [comment]
code is ok, but CI is not passing :(

**Pull request merged**